dev/bazel/deps/mpi.bzl

```diff
@@ -0,0 +1,104 @@
+#===============================================================================
+# Copyright 2020-2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#===============================================================================
+
+load("@onedal//dev/bazel:repos.bzl", "repos")
+
+mpi_repo = repos.prebuilt_libs_repo_rule(
+    includes = [
+        "include",
+    ],
+    libs = [
+        "lib/release/libmpi.so",
+        "lib/release/libmpi.so.12",
+        "lib/release/libmpi.so.12.0",
+        "lib/release/libmpi.so.12.0.0",
+        "libfabric/lib/libfabric.so",
+        "libfabric/lib/libfabric.so.1",
+        "libfabric/lib/prov/libefa-fi.so",
+        "libfabric/lib/prov/libmlx-fi.so",
+        "libfabric/lib/prov/libpsmx2-fi.so",
+        "libfabric/lib/prov/librxm-fi.so",
+        "libfabric/lib/prov/libshm-fi.so",
+        "libfabric/lib/prov/libsockets-fi.so",
+        "libfabric/lib/prov/libtcp-fi.so",
+        "libfabric/lib/prov/libverbs-fi.so",
```

Comment:
Can it automatized?

Reply:
I think yes, but requires engineering in `prebuilt_libs_repo_rule`

Reply:
I was wrong, this is easy to fix, will do